Dockerfile

```diff
@@ -1,23 +1,29 @@
-FROM python:3.8-alpine as base
-RUN apk add --no-cache curl tini && \
-    adduser -u 1000 -g 1000 -D kyaku
-WORKDIR /home/kyaku/renku-notebooks
-
-FROM base as builder
-ENV POETRY_HOME=/opt/poetry
+FROM python:3.11-bullseye as builder
```

Comment:
python 3.10 is used elsewhere - is that on purpose?

Reply:
@rokroskar the other components install the renku cli. And I think the CLI is still not fully compatible with python 3.11. This one does not use the CLI so we can go to 3.11.

Reply:
if you don't need 3.11 we could keep it consistent?

Reply:
Ok cool I will update to 3.11.

Reply:
oh it does? then we can build 3.11 base images as well?

Reply:
The problem I had was that I could not get numpy installed on the debian images I was using with python 3.11. But now since we removed numpy from the dependencies things work.